app/views/govuk_publishing_components/components/_attachment.html.erb

```diff
@@ -0,0 +1,56 @@
+<%
+  attachment = GovukPublishingComponents::Presenters::Attachment.new(attachment)
+  target ||= nil
+  hide_help_text ||= false
+  attributes = []
+
+  if attachment.content_type_abbr
+    content = tag.abbr(attachment.content_type_abbr,
+                       class: "gem-c-attachment__abbr",
+                       title: attachment.readable_content_type)
+    attributes << tag.span(content)
+  end
+
+  if attachment.readable_file_size
+    attributes << tag.span(attachment.readable_file_size)
+  end
+
+  if attachment.readable_number_of_pages
+    attributes << tag.span(attachment.readable_number_of_pages)
+  end
+%>
+<%= tag.section class: "gem-c-attachment" do %>
+  <%= tag.div class: "gem-c-attachment__thumbnail" do %>
+    <%= link_to attachment.url, target: target do %>
```

Comment:
This link is focusable but doesn't have the correct focus styles set, it just has the browser default.

![Screen Shot 2019-05-07 at 09 25 53](https://user-images.githubusercontent.com/861310/57285005-38ba5e80-70aa-11e9-88ae-c0b98415e707.png)

Having thought about it, it's probably not worth letting this link be focusable - it's the same as the text link right? Maybe just give it a `tabindex=-1` attribute, people don't need to tab through two identical links.

Reply:
I've added `tabindex=1`